docs/resources/technical-glossary.md

```diff
@@ -0,0 +1,309 @@
+**Byzantine Faults (Double signing)**
+
+A validator is said to make a byzantine fault when they sign conflicting messages/blocks at the same height and round.
+
+
+#### C
+
+**Chain ID**
+A unique identifier for the blockchain. Different prefixes of the Chain ID are used to distinguish between _mainnet_, _devnet_, and _testnet_. 
+For example, the Chain ID of our [testnet](https://crypto.org/docs/getting-started/croeseid-testnet.html) is `testnet-croeseid-2`.
+
+**Chain-abci**
+
+The **A**pplication **B**lock**C**hain **I**nterface connects Tendermint (for consensus operations) to the actual applications.
+
+**Chain-maind**
+
+chain-maind is the all-in-one command-line interface. It supports wallet management, funds transfer, and staking operations. 
+ 
+**Client-rpc**
+
+The JSONRPC interface of the wallet client. It can be used to be integrated with different services and also power the Sample Wallet. It provides nearly the same set of operations as Client-cli does.
+
+**Commission Max Change Rate**
+
+The maximum daily increase of the validator’s commission. This parameter cannot be changed by the validator operator.
+
+**Consensus**
+
+The process of a group of entities to agree on a particular data value, and there are a variety of algorithms used for determining consensus.
+
+**CosmWasm**
+
+It is a new smart contracting platform built for the cosmos ecosystem. It is written as a module that can plug into the Cosmos SDK. 
+
+**Create-validator(Transaction Type: `create-validator`)**
+
+The transaction for joining the network as a validator.
+
+
+#### D
+
+**Delegate (Transaction type: `delegate`)**
+
+The transaction that transfers funds from a transfer address to a staking address for staking purposes. Delegators should perform due diligence on validator candidates before delegating, as well as spreading their stake over multiple validators. It will also help keep the network decentralized.
+
+**Delegator**
+
+CRO holders who do not run validator operations themselves. Users can delegate CRO to validators and obtain a part of its revenue in exchange. Because they share revenue with their validators, delegators also share responsibility. Should a validator misbehave, each of its delegators will be partially slashed in proportion to their stake. 
+
+
+#### F
+
+**[Faucet](https://crypto.org/faucet)**
+
+Service carried out that dispenses funds in the form of free test ether that can be used on a testnet.
+
+**Full Node**
+
+A full node is a node that participates in the network but will not help secure it. Full nodes can be used to store the entire state of a blockchain. There are two forms of state for Tendermint. First, blockchain state, this represents the blocks of a blockchain. Secondly, there is the Application state, this represents the state that transactions modify. 
+
+
+#### G
+
+**Genesis**
+
+The origin of a blockchain, also known as block 0. It can also be used to reference the initial state of the blockchain at origination.
```

Comment:
```suggestion
The origin of a blockchain, also known as block 1. It can also be used to reference the initial state of the blockchain at origination.
```

Reply:
can also add the mainnet genesis as an example: https://github.com/crypto-org-chain/mainnet/blob/main/crypto-org-chain-mainnet-1/genesis.json